src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
-
+spring.datasource.url=jdbc:mysql://localhost:3306/emp_db
```

Comment:
have common application properties so dont commit this

Reply:
### Done